client/src/components/Community/Community.css

```diff
@@ -10,7 +10,8 @@
 }
 
 .community-component .union-img {
-  height: 250vh;
+  margin-top: 20vh;
```

Comment:
I think its not lining up with the corresponding title. This looks so unsatisfying to me..

Reply:
Not clear, Can uh ellaborate further !!

Reply:
The circle of `union.svg` is not lining up with "From the Geeks' Satellite.." which is what the design aims for. I think achieving that alignment is more fundamental aspect of the design.

Reply:
Okay, Got it :-)